lib/stat_tracker.rb

```diff
@@ -61,95 +61,72 @@ def highest_total_score
 
   def lowest_total_score
     score_sum = 10000 #need to update for csv file with no data
-    contents = CSV.open(@game_path, headers: true, header_converters: :symbol)
-    contents.each do |row|
-      away_goals = row[:away_goals].to_i
-      home_goals = row[:home_goals].to_i
-      if score_sum > (away_goals + home_goals)
-        score_sum = (away_goals + home_goals)
```

Comment:
great way on getting rid of extra lines. you even took away unnecessary assignments!